packages/esm-appointments-app/src/index.ts

```diff
@@ -78,6 +79,14 @@ export const appointmentsList = getSyncLifecycle(appointmentsListComponent, opti
 
 export const earlyAppointments = getSyncLifecycle(earlyAppointmentsComponent, options);
 
+export const editAppointmentsForm = getSyncLifecycle(appointementsForm, options);
+
+export const searchPatient = getSyncLifecycle(patientSearch, options);
+
+export const createAppointment = getSyncLifecycle(appointementsForm, options);
+
+export const addAppointment = getSyncLifecycle(appointementsForm, options);
```

Comment:
Defer to @brandones  but wouldn't it make sense to the just create a single "appointmentForm" here instead of "editAppointmentForm", "createAppointment" and "addAppointment" and then just that same component when registering the three different workspaces below?

Reply:
Yeah that would be great, thanks @mogoodrich , thanks @kb019